packages/react/src/patterns/blocks/FeatureCardBlockLarge/__stories__/FeatureCardBlockLarge.stories.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import './index.scss';
+import { object, text, withKnobs } from '@storybook/addon-knobs';
+import { ArrowRight20 } from '@carbon/icons-react';
+import FeatureCardBlockLarge from '../FeatureCardBlockLarge';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|FeatureCardBlockLarge', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const cardheading = text(
+      'Card heading(required):',
```

Comment:
We're starting to add the actual prop name in parenthesis next to the description, can you update to something like : 
```
const cardheading = text('Card heading(required) (heading):', 'Explore AI use cases in all industries');
```

```